### ISSUE TITLE
Unable to authorise account 

### ISSUE THREAD

**Issue by ChiefWarPig** [MCXboxBroadcast/Broadcaster#154]
Hi I am getting the following message every time I try to re connect this account. I am reconnecting due to Geyser being reinstalled. I have logged into the account on Minecraft and Microsoft website and no actions their side I can see being requested. How do I go about fixing this issue please? 
![Screenshot 2024-10-14 121228](https://github.com/user-attachments/assets/1f270b08-0a2b-4dbd-b7ed-ad0d6d371632)

**Comment by rtm516** [comment]
How are you logging in? Password less login can sometimes cause this. Try with a full passworded login

**Comment by ChiefWarPig** [comment]
I am not given an option to use a password when I log in. If I click other options it just tries to make me select from receiving an email or text message code to login and authorise?

**Comment by rtm516** [comment]
There should be an option for it, please can you provide a screenshot of what your seeing?

**Comment by ChiefWarPig** [comment]
Hi I have managed to sort it. So when I was trying to connect I copied the code the plugin provides inside the console and then it takes you to a Microsoft authorisation page which didn't/doesn't offer password authentication it was trying to force me to receive a code via text or email which seems to be what was causing the issue. I have changed the password on the account, selected stay logged in and setup pin verification which seems to have done the trick. 
Thank you for the support.
